Kitodo/src/main/java/de/sub/goobi/persistence/apache/FolderInformation.java

```diff
@@ -63,57 +63,46 @@ public URI getImagesTifDirectory(boolean useFallBack) {
         /* nur die _tif-Ordner anzeigen, die nicht mir orig_ anfangen */
         FilenameFilter filterDirectory = new FileNameEndsAndDoesNotBeginWithFilter(DIRECTORY_PREFIX + "_",
                 "_" + DIRECTORY_SUFFIX);
-        URI tifOrdner = null;
+        URI tifDirectory = null;
         ArrayList<URI> directories = fileService.getSubUris(filterDirectory, dir);
         for (URI directory : directories) {
-            tifOrdner = directory;
+            tifDirectory = directory;
         }
 
-        if (tifOrdner == null && useFallBack) {
+        if (tifDirectory == null && useFallBack) {
             String suffix = ConfigCore.getParameter("MetsEditorDefaultSuffix", "");
             if (!suffix.equals("")) {
-                tifOrdner = iterateOverDirectories(dir, suffix);
+                tifDirectory = iterateOverDirectories(dir, suffix);
             }
         }
-        if (!(tifOrdner == null) && useFallBack) {
+        if (!(tifDirectory == null) && useFallBack) {
             String suffix = ConfigCore.getParameter("MetsEditorDefaultSuffix", "");
             if (!suffix.equals("")) {
-                URI tif = tifOrdner;
+                URI tif = tifDirectory;
                 ArrayList<URI> files = fileService.getSubUris(tif);
                 if (files == null || files.size() == 0) {
                     ArrayList<URI> folderList = fileService.getSubUris(dir);
                     for (URI folder : folderList) {
                         if (folder.toString().endsWith(suffix)) {
-                            tifOrdner = folder;
+                            tifDirectory = folder;
                             break;
                         }
                     }
                 }
             }
         }
 
-        if (tifOrdner == null) {
-            tifOrdner = URI.create(this.title + "_" + DIRECTORY_SUFFIX);
+        if (tifDirectory == null) {
+            tifDirectory = URI.create(this.title + "_" + DIRECTORY_SUFFIX);
         }
 
-        URI rueckgabe = getImagesDirectory().resolve(tifOrdner);
+        URI result = getImagesDirectory().resolve(tifDirectory);
 
-        if (!rueckgabe.toString().endsWith(File.separator)) {
-            rueckgabe = rueckgabe.resolve(File.separator);
+        if (!result.toString().endsWith(File.separator)) {
+            result = result.resolve(File.separator);
         }
 
-        return rueckgabe;
-    }
-
-    /**
-     * Get tif directory exists.
-     *
-     * @return true if the Tif-Image-Directory exists, false if not
-     */
-    public Boolean getTifDirectoryExists() {
```

Comment:
You can not remove method `getTifDirectoryExists()` as this method is still used in `Kitodo/src/main/webapp/pages/inc_Prozessverwaltung/Prozesse_Liste.xhtml` lines 468, 476, 483, 490, 496 and 503!

Reply:
item.tifDirectoryExists - item is process object, so this method should be in ProcessDTO:

`<t:dataTable id="auflistung" styleClass="standardTable" width="100%" cellspacing="1px" cellpadding="1px"
                 headerClass="standardTable_Header"
                 rowClasses="standardTable_Row1,standardTable_Row2,standardTable_Row1,standardTable_Row2" var="item"
                 value="#{ProzessverwaltungForm.page.listReload}">`

Reply:
Then this method must be available in `ProcessDTO` which is not the case yet.

Reply:
Actually it is:

public boolean isTifDirectoryExists() {
        return tifDirectoryExists;
    }

As it is boolean, it works with is....: https://stackoverflow.com/questions/14400222/boolean-properties-starting-with-is-does-not-work

Reply:
Ah, ok. Thank you for this explanation. Then this method could be removed.